tests/unit/resources/test_population.py

```diff
@@ -28,25 +31,37 @@ def population_tester(population):
 
 
 @responses.activate
-def test_should_get_population(ENDPOINT_BASE, population, populations):
+def test_should_get_population(client, ENDPOINT_BASE, population, populations):
     responses.add(
         responses.GET,
         ENDPOINT_BASE + f"/experiments/{EXP_ID}/populations/{population._id}",
         json=populations[0],
     )
-    pop = Population.get(EXP_ID, population._id)
+    pop = client.get_population(EXP_ID, population._id)
+    assert pop.unique_name is None
```

Comment:
Same Q about test coverage.

(Also don't really need to assert the absence of `pop.unique_name`. We might add it to the GET route sometime even.)

Reply:
:+1: